service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+    
```

Comment:
Can we rename the file to OrderService ?

Reply:
completed